prusti-encoder/src/encoders/type/domain.rs

```diff
@@ -63,21 +63,30 @@ pub enum DiscrBounds<'vir> {
 
 #[derive(Clone, Copy, Debug)]
 pub enum DomainEncSpecifics<'vir> {
-    Param,
     Primitive(DomainDataPrim<'vir>),
     // structs, tuples
     StructLike(DomainDataStruct<'vir>),
-    EnumLike(Option<DomainDataEnum<'vir>>)
+    EnumLike(Option<DomainDataEnum<'vir>>),
+    Param,
```

Comment:
The `Param` here is to indicate that a generic type was passed to `DomainEnc` in the `DomainEncSpecifics`. Unrelated to formatting changes.

Reply:
Oh the reason I'm asking is because the diff shows that the `Param` variant was moved to be the last one declared instead of the first one, but it already existed.

Reply:
Ah now I remember, I think what happened was I deleted it because I thought it was no longer necessary, and then I re-added it (at the bottom)